lib/kinde_sdk/kinde_client_sdk.ex

```diff
@@ -250,22 +221,8 @@ defmodule KindeClientSDK do
   """
   @spec register(
           Plug.Conn.t(),
-          %KindeClientSDK{
-            additional_params: map,
-            auth_status: atom,
-            authorization_endpoint: <<_::64, _::_*8>>,
-            cache_pid: pid,
-            client_id: any,
-            client_secret: any,
-            domain: binary,
-            grant_type: any,
-            logout_endpoint: <<_::56, _::_*8>>,
-            logout_redirect_uri: any,
-            redirect_uri: binary,
-            scopes: any,
-            token_endpoint: <<_::64, _::_*8>>
-          },
-          map
+          map(),
```

Comment:
Looks good but to be more precise with the spec like it was before

we can use

`KindeClientSDK.t()`

and declate it

`  @type t :: %KindeClientSDK{
    additional_params: map,
    auth_status: atom,
    authorization_endpoint: binary,
    cache_pid: pid,
    client_id: any,
    client_secret: any,
    domain: binary,
    grant_type: any,
    logout_endpoint: binary,
    logout_redirect_uri: any,
    redirect_uri: binary,
    scopes: any,
    token_endpoint: binary
  }`

Reply:
Thanks @imahmedismail, once the changes have been made. We will merge

Reply:
Done with the changes
cc: @onderay @femz12